### ISSUE TITLE
Incorrect name and no state is defined for the ‘Hamburger’ menu button.

### ISSUE THREAD

**Issue by msftedad** [docsifyjs/docsify#2599]
### Description

**User Impact:**
Screen reader users will be impacted as they are not able to understand the purpose and its relationship with the content and hence will not be able to navigate or interact with it.

**Refer Attachment:**
1. Incorrect name and no state is defined for the ‘Hamburger’ menu button.png
2. Incorrect name and no state is defined for the ‘Hamburger’ menu button.mp4

<img width="960" height="608" alt="Image" src="https://github.com/user-attachments/assets/af05386d-2a82-4ec6-90d6-3a7ce7998f43" />
https://github.com/user-attachments/assets/02b2275b-7b3e-48c5-b6a6-a6a66a169d97

### Expected behavior

Screen reader should announce as 'Hamburger menu button collapsed' on using tab key.

### Actual behavior

Incorrect name and no state is defined for the ‘Hamburger’ menu button.
Screen reader is announcing as 'Menu button'.

### Steps to reproduce

1. Open the URL: https://docs.pwabuilder.com/#/builder/android in latest edge browser.
2. PWA Builder - Packaging page will be displayed.
3. Press tab key to reach 'Hamburger menu button'.
4. Verify using tab key whether screen reader is announcing correct name and state for 'Hamburger menu button' or not.

### Environment

OS: Windows 11 Enterprise Insider Preview Version 24H2 (OS Build 26100.3775)
Browser Microsoft Edge Version 135.0.3179.85 (Official build) (64-bit)
Screen Reader: Narrator, NVDA (2024.4.1.34550), JAWS (2025.2412.50.400),
URL: https://docs.pwabuilder.com/#/builder/android

### Additional Information

- [ ] Bug still occurs when all/other plugins are disabled?
